From 16f73174a5fc3ffb558a830ba29489b60cbebc90 Mon Sep 17 00:00:00 2001
From: Wallace Wang <r59996@freescale.com>
Date: Fri, 27 Jun 2008 16:08:39 +0800
Subject: [PATCH 028/435] ENGR00081249 The converted stream's sound has noise

The /tmp dirctory size is limited to 512 while the unit
test needs up to 20M memory size
Use /dev to replace /tmp for temp storage.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 test/mxc_asrc_test/autorun-asrc.sh | 22 +++++++++++-----------
 test/mxc_asrc_test/mxc_asrc_test.c |  2 +-
 2 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/test/mxc_asrc_test/autorun-asrc.sh b/test/mxc_asrc_test/autorun-asrc.sh
index ba130c9..17ccbc6 100755
--- a/test/mxc_asrc_test/autorun-asrc.sh
+++ b/test/mxc_asrc_test/autorun-asrc.sh
@@ -11,47 +11,47 @@ check_devnode "/dev/mxc_asrc"
 check_devnode "/dev/snd/pcmC0D0p"
 
 if [ "$STATUS" = 0 ]; then
-run_testcase " ./mxc_asrc_test.out -to 48000 audio8k16S.wav /tmp/s48.wav"
+run_testcase " ./mxc_asrc_test.out -to 48000 audio8k16S.wav /dev/s48.wav"
 fi
 
 if [ "$STATUS" = 0 ]; then
-run_testcase " aplay -N -M /tmp/s48.wav"
+run_testcase " aplay -N -M /dev/s48.wav"
 fi
 
 if [ "$STATUS" = 0 ]; then
-run_testcase " ./mxc_asrc_test.out -to 96000 /tmp/s48.wav /tmp/s96.wav"
+run_testcase " ./mxc_asrc_test.out -to 96000 /dev/s48.wav /dev/s96.wav"
 fi
 
 if [ "$STATUS" = 0 ]; then
-run_testcase " aplay -N -M /tmp/s96.wav"
+run_testcase " aplay -N -M /dev/s96.wav"
 fi
 
 if [ "$STATUS" = 0 ]; then
-run_testcase " rm /tmp/s48.wav"
+run_testcase " rm /dev/s48.wav"
 fi
 
 if [ "$STATUS" = 0 ]; then
-run_testcase " ./mxc_asrc_test.out -to 88200 /tmp/s96.wav /tmp/s882.wav"
+run_testcase " ./mxc_asrc_test.out -to 88200 /dev/s96.wav /dev/s882.wav"
 fi
 
 if [ "$STATUS" = 0 ]; then
-run_testcase " aplay -N -M /tmp/s882.wav"
+run_testcase " aplay -N -M /dev/s882.wav"
 fi
 
 if [ "$STATUS" = 0 ]; then
-run_testcase " rm /tmp/s96.wav"
+run_testcase " rm /dev/s96.wav"
 fi
 
 if [ "$STATUS" = 0 ]; then
-run_testcase " ./mxc_asrc_test.out -to 44100 /tmp/s882.wav /tmp/s441.wav"
+run_testcase " ./mxc_asrc_test.out -to 44100 /dev/s882.wav /dev/s441.wav"
 fi
 
 if [ "$STATUS" = 0 ]; then
-run_testcase " aplay -N -M /tmp/s441.wav"
+run_testcase " aplay -N -M /dev/s441.wav"
 fi
 
 if [ "$STATUS" = 0 ]; then
-run_testcase " rm /tmp/s882.wav /tmp/s441.wav /tmp/raw.txt"
+run_testcase " rm /dev/s882.wav /dev/s441.wav /dev/raw.txt"
 fi
 print_status
 exit $STATUS
diff --git a/test/mxc_asrc_test/mxc_asrc_test.c b/test/mxc_asrc_test/mxc_asrc_test.c
index d55ee2b..3e0c781 100644
--- a/test/mxc_asrc_test/mxc_asrc_test.c
+++ b/test/mxc_asrc_test/mxc_asrc_test.c
@@ -385,7 +385,7 @@ int main(int ac, char *av[])
 	if (err < 0)
 		goto end_err;
 
-	fd_raw = fopen("/tmp/raw.txt", "wb+");
+	fd_raw = fopen("/dev/raw.txt", "wb+");
 	/* Config HW */
 	err += play_file(fd_asrc, fd_dst, fd_raw, &audio_info);
 
-- 
1.8.0

